Marlin/temperature.cpp

```diff
@@ -1736,7 +1744,16 @@ void Temperature::isr() {
         #define GE0 >=
       #endif
       if (current_temperature_raw[0] GE0 maxttemp_raw[0]) max_temp_error(0);
-      if (minttemp_raw[0] GE0 current_temperature_raw[0]) min_temp_error(0);
+      if (minttemp_raw[0] GE0 current_temperature_raw[0] && !is_preheating(0) && target_temperature[0] > 0.0f) {
```

Comment:
Does the addition of the condition `target_temperature[0] > 0.0f` fix a bug? Should we be doing this already?